src/hooks/useSyncChainQuery.ts

```diff
@@ -37,9 +37,14 @@ export default function useSyncChainQuery() {
       selectChain(urlChainId)
     }
     // If a user has a connected wallet and has manually changed their chain, delete the old chain query param
-    else if (account && urlChainId && chainIdRef.current !== chainId && chainId !== urlChainId) {
-      searchParams.delete('chain')
-      setSearchParams(searchParams)
+    else if (account && chainIdRef.current !== chainId && chainId !== urlChainId) {
```

Comment:
`else if` should be `} else if` - not broken across two lines.

Reply:
I placed this a line below intentionally because I had the comment explaining above it. It reads more clearly imo than having `} else if` on the same line and having the comment be inside of the `if` block

Reply:
I completely agree with you but thought that lint disallowed this. interesting!